### PR TITLE
Fix API Sample tests

### PR DESCRIPTION
**Pull request by mhegazy**
We were generating typescript_standalone.d.ts by converting `namespace ts` to `module "typescript"` that seems to be broken with nested namespaces. instead of trying to fix the regexp, i have switched to use the `typescriptservcies.d.ts`

Also removing the generation portion from jake.

### CODE REVIEW

**Comment by weswigham** [comment]
Reminder: Those nested namespaces are `@internal` declarations that shouldn't be in the `.d.ts` in the first place, and the `standalone` definition file we still build so people can, eg, use that declaration file with older versions of typescript in global script projects. I'm pretty OK with removing it for the `3.0` release, though - seeing a umd declaration appended to the bottom of the services `.d.ts` would have the same effect for any recent typescript release (and we use even newer type-syntactic features anyway).

File: Jakefile.js [hunk 1]
```diff
@@ -139,11 +139,6 @@ compileOutputConfigFile('src/typescriptServices/tsconfig.json', [], [copyright],
     // Created by appending 'export = ts;' at the end of the standalone file to turn it into an external module
     var nodeDefinitionsFileContents = definitionFileContents + "\nexport = ts;";
```
Comment (weswigham) [inline Jakefile.js]: To support browser cases seamlessly, this should also probably append `"\nexport as namespace ts;"` now.

Comment (mhegazy) [reply Jakefile.js]: fair point.

Comment (mhegazy) [reply Jakefile.js]: I can get these done in a different PR.

**Pull request merged**